libparsec/crates/client_connection/Cargo.toml

```diff
@@ -43,8 +43,16 @@ log = { workspace = true }
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 # TODO: support SSE for Web
 reqwest-eventsource = { workspace = true }
+tokio = { workspace = true, features = ["net"] }
 
 [dev-dependencies]
 libparsec_tests_lite = { workspace = true }
 # Note `libparsec_tests_fixtures` enables our `test-with-testbed` feature
 libparsec_tests_fixtures = { workspace = true, features = ["test-with-client-connection-testbed"] }
+# FIXME: `rstest` is unstable when renamed. https://github.com/la10736/rstest/issues/221
+rstest = { workspace = true }
+futures-util = { workspace = true }
+rand = { workspace = true }
+anyhow = { workspace = true }
+log = { workspace = true }
```

Comment:
use print instead of log:
- in test you don't need fancy level of output: it's either the test pass and the whole output is discarded, or the test fail and we want all logs
- using log is error prone (nothing will show up if the correct level is not configured, which is not trivial to do given nothing explains you should use the `LOG_LEVEL` environ variable)
- using print force to think a bit more of the logs that should stay (with logs it is tempting to have plenty of logs of low level "just in case" which in the end are never used, or are set at a higher level and clutter everything)
- using log is probably not going to work in the long run: I plan to use the event bus to pass the logs in the application (this has the benefit of 1. being testable and 2. simplify passing logs to the GUI for Sentry)

Reply:
done